build/MUX-PR.yml

```diff
@@ -25,11 +25,12 @@ stages:
   - job: Build
     condition: eq(variables['useBuildOutputFromBuildId'],'')
     pool: 
-      ${{ if eq(variables['System.CollectionUri'], 'https://dev.azure.com/ms/') }}:
+      ${{ if eq(variables['System.CollectionUri'], 'https://dev.azure.com/shine-oss/') }}:
         name: WinDevPoolOSS-L
-      ${{ if ne(variables['System.CollectionUri'], 'https://dev.azure.com/ms/') }}:
+      ${{ if ne(variables['System.CollectionUri'], 'https://dev.azure.com/shine-oss/') }}:
         name: WinDevPool-L
-      demands: ImageOverride -equals WinDevVS17-latest
+      demands: 
+        - ImageOverride -equals WinDevVS17-latest
```

Comment:
Did you really intend to make this change?   It's not consistent with the build/MUX-CI.yml file for instance.

Reply:
it's just an alternative syntax. This syntax lets you add multiple demands (one per line). I was using the version override demand during testing.